lib/actions.js

```diff
@@ -58,6 +66,64 @@ const _createAccount = (walletId, accountId, accountInfo) => ({
   type: WALLET_CREATE_ACCOUNT,
   payload: { walletId, accountId, accountInfo }
 });
+// multisig wallet helpers
+const _getMultisigWallets = (multisigWallets) => ({
+  type: SET_MULTISIG_WALLETS,
+  payload: { multisigWallets }
+});
+const _selectMultisigWallet = walletId => ({
+  type: USER_SELECT_MULTISIG_WALLET,
+  payload: { walletId }
+});
+const _selectMultisigProposal = proposalId => ({
+  type: USER_SELECT_MULTISIG_PROPOSAL,
+  payload: { proposalId }
+});
+const _setMultisigWalletInfo = (walletId, multisigWalletInfo) => ({
+  type: SET_MULTISIG_WALLET_INFO,
+  payload: { walletId, multisigWalletInfo }
+});
+const _setMultisigProposal = (walletId, proposals) => ({
+  type: SET_MULTISIG_WALLET_PROPOSALS,
+  payload: { walletId, proposals }
+});
+const _setMultisigProposalMTX = (walletId, proposalId, mtx) => ({
+  type: SET_MULTISIG_PROPOSAL_MTX,
+  payload: { walletId, proposalId, mtx }
+});
+const _selectXPUB = xpub => ({
+  type: USER_SELECT_XPUB,
+  payload: { selectedXPUB: xpub }
+});
+
+export const selectXPUB = xpub => async dispatch => {
+  dispatch(_selectXPUB(xpub));
+}
+
+export const selectMultisigProposal = proposalId => async dispatch => {
```

Comment:
this could just be the normal action creator format that you have in `_selectMultisigProposal` and then connected to the state's dispatcher via `connect` and `mapDispatchToProps`